src/shared/domain/reportInteraction.ts

```diff
@@ -21,7 +23,45 @@ function getExtraProperties() {
   return extraProperties;
 }
 
-export function reportInteraction(interactionName: string, properties?: Record<string, unknown>) {
+// hey future dev: don't forget to add any new value to our features tracking dashboard!
+export type INTERACTION_NAME =
```

Comment:
In other places we use CamelCases for type names. Any reason why this should be different?

Reply:
No actually, I'll fix it. Thx!